js/src/components/PostPublish.js

```diff
@@ -20,11 +20,15 @@ export default function PostPublish( { permalink } ) {
 			<li>
 				<a href={ "https://www.facebook.com/sharer/sharer.php?u=" + encodedUrl } target="_blank" rel="noopener noreferrer">
 					<FacebookIcon />
+					{ __( "Facebook", "wordpress-seo" ) }
+					<span className="screen-reader-text">{ __( "(Opens in a new browser tab)", "yoast-components" ) }</span>
```

Comment:
Not sure the "yoast-components" text domain is appropriate. I guess it should be "wordpress-seo".

Reply:
`"yoast-components"` seems to be the domain used by `makeOutboundLink` which we are mimicking.
I agree that using that is something that doesn't happen at all in the plugin, I will fix that.